### PR TITLE
Update for issue #1347 - Update automation logging to work better

### PR DESCRIPTION
**Pull request by netniV**
Logging of api_automation.php now records more items using the standard expected by clog functionality.  Also adds Graph Template's to the items translated plus adds a couple of missing items to the no_http_headers global array so debug logging can be assigned.

### CODE REVIEW

File: lib/clog_webapi.php [hunk 1]
```diff
@@ -744,3 +749,72 @@ function clog_regex_rra($matches) {
...
+
+		$result .= $matches[1] . "<a href='";
+
+		$queryDescriptions = array();
+		foreach ($querys as $query) {
+			$queryDescriptions[$query['id']] = html_escape($query['title']);
+		}
+
+		$i=0;
+		foreach ($query_ids as $query_id) {
```
Comment (cigamit) [inline lib/clog_webapi.php]: Need to check for sizeof() here.  Don't assume that if you have id's, that by the time you view the log, they will still be there.

File: lib/api_automation.php [hunk 2]
```diff
@@ -2753,15 +2760,16 @@ function automation_debug($text) {
 	global $debug;
 	static $message = '';
```
Comment (cigamit) [inline lib/api_automation.php]: So, are you saying that we will always be in debug mode?

File: lib/clog_webapi.php (hunk 1 above)
Comment (netniV) [reply lib/clog_webapi.php]: Good catch, I think that's a problem for all the regex functions so I'll check them all.

File: lib/api_automation.php (hunk 2 above)
Comment (netniV) [reply lib/api_automation.php]: No, it picks up $debug from poller_automation.php and operates more in line with the rest of the system when it comes to using a global $debug.  The -d debug flag will set the global $debug to true which in turn updates the POLLER_VERBOSITY level used in the automation_debug()'s cacti_log call from HIGH (only if the system is set to high) to NONE (always print)

File: lib/clog_webapi.php (hunk 1 above)
Comment (netniV) [reply lib/clog_webapi.php]: I have now commited https://github.com/Cacti/cacti/pull/1349/commits/4f606895f274b35fe06a3e19fdd8f4183126340e with additional fix https://github.com/Cacti/cacti/pull/1349/commits/d1da63028e26710e30fdec1f3e035dcf2bb23105 for the CI error that should address the sizeof() request

**Comment by netniV** [comment]
With the extension changes here, this may be one to put into 1.2 so more testing is done.

**Comment by cigamit** [comment]
Agree.  Let's just sit on this one for a bit.

**Comment by netniV** [comment]
Since 1.1.x is now retired.  Shall we merge this into 1.2 and see what happens?

**Pull request closed without merging**